augur/application/cli/backend.py

```diff
@@ -38,8 +39,15 @@ def cli():
 def start(disable_collection):
     """Start Augur's backend server."""
 
-    raise_open_file_limit(100000)
+    try:
+        raise_open_file_limit(100000)
+    except Exception as e: 
```

Comment:
@sgoggins or @IsaacMilarky: Can you catch the specific exception instead of the base Exception class?

Reply:
@ABrain7710 : I don't know what the specific exception was ... @IsaacMilarky : Do you recall?